scripts/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then 
+    BUILD_TYPE=Release
+else 
+    BUILD_TYPE="$1"
+fi
+
+PWD=$(pwd)
```

Comment:
See below, I use a git command to get us to the top of the root directory, this is merely so that we take note of wherever the user is executing from and return them to that spot when they are done.